### PR TITLE
Add tree-sitter-query

### PR DESCRIPTION
**Pull request by Will-Sommers**
Prepping to support scheme as perhaps the first language using scheme queries itself. One thing to note is that this is not one of the langues within the [language identifier](https://code.visualstudio.com/docs/languages/identifiers) list. However, a breakpoint here shows that [`languageId`](https://github.com/cursorless-dev/vscode-parse-tree/blob/main/src/extension.ts#L93) returns `"scheme"`. I have a scheme plugin installed so perhaps that is adding the `.scm` filetype to the supported languages of my local VSCode.

### CODE REVIEW

**Comment by pokey** [comment]
Don't we want to use https://github.com/nvim-treesitter/tree-sitter-query instead?

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
 		"onLanguage:shellscript",
 		"onLanguage:typescript",
 		"onLanguage:sparql",
+		"onLanguage:scm",
```
Comment (Will-Sommers) [inline package.json:49]: `languageId` is `scm` but the nvim-treesitter package is called `query`.

image

**Comment by Will-Sommers** [comment]
Yep @pokey! Updated.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # For generating .wasm files for parsers
 # See https://www.npmjs.com/package/web-tree-sitter
-LANGUAGES = agda c clojure cpp css c-sharp bash go haskell html java javascript json latex markdown php python ruby rust scala scss sparql tsx typescript yaml
+LANGUAGES = agda bash c c-sharp clojure cpp css go haskell html java javascript json latex markdown php python query ruby rust scala scss sparql tsx typescript yaml
```
Comment (pokey) [inline Makefile:3]: `"sort every paint tail gust"` 🙌😊

**Review by pokey — APPROVED** [review]
looks good thanks!!

**Comment by pokey** [comment]
shipped!

**Pull request merged**